### PR TITLE
koord-manager: Fix log print fmt and add an example for slo-controller-config

### PR DESCRIPTION
**Pull request by LeoLiuYan**
Signed-off-by: LeoLiuYan <929908264@qq.com>

### Ⅰ. Describe what this PR does

As the title says.

### Ⅱ. Does this pull request fix one issue?

NONE.

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/slo-controller/config/constants.go [hunk 1]
```diff
@@ -33,5 +33,94 @@ Koordinator uses configmap to manage the configuration of SLO, the configmap is
...
+          }
+        }
+      ]
+    }
+  resource-threshold-config: |
+    {
+      "clusterStrategy": {
+        "enable": true,
+        "cpuSuppressThresholdPercent": 75,
+        "cpuSuppressPolicy": "cpuBurst",
```
Comment (saintube) [inline pkg/slo-controller/config/constants.go:99]: the example is recommended to be valid. here the value of `cpuSuppressPolicy` may be `cpuset`

File: pkg/slo-controller/config/constants.go [hunk 2]
```diff
@@ -33,5 +33,94 @@ Koordinator uses configmap to manage the configuration of SLO, the configmap is
...
+        {
+          "nodeSelector": {
+            "matchLabels": {
+              "kubernetes.io/kernel": "alios"
+            }
+          },
+          "lsClass": {
+            "memoryQOS": {
+              "enable": true,
+              "wmarkRatio": 50,
```
Comment (saintube) [inline pkg/slo-controller/config/constants.go:80]: Actually, this wmarkRatio is too small for LS pods. How about using some of the recommended values defined in `DefaultResourceQOSStrategy()`

Comment (LeoLiuYan) [reply pkg/slo-controller/config/constants.go]: Review again.  @saintube

**Review by jasonliu747 — APPROVED** [review]

**Pull request merged**